김진/231206/문자열-다루기-기본.js

```diff
@@ -0,0 +1,11 @@
+function solution(s) {
+  if (String(s).length === 4 || String(s).length === 6) {
+    for (let i = 0; i < String(s).length; i++) {
```

Comment:
```suggestion
    for (let i = 0; i < s.length; i++) {
```
s는 애초에 문자열이라 String을 쓸 이유는 없는 것 같아요